site/content/en/docs/integration/toloka.md

```diff
@@ -0,0 +1,213 @@
+
+   ![Toloka Project Configuration](/images/toloka02.png)
+
+   - **Provider**: Choose Toloka as your provider.
+   - **Environment**: Select either **Sandbox** for a testing environment or **Production** for a live environment.
+   - **Authorization key**: Enter the [Requester Authentication Key](https://toloka.ai/docs/toloka-kit/registration/).
+   - **Project description** (Optional): Provide a brief description of your project.
+
+4. Click **Submit**.
+   A pop-up indicating **Toloka Project Created** will appear, along with the **Update project** form.
+
+   ![Project Update Form](/images/toloka03.png)
+
+   - ![Open Project in Toloka](/images/toloka04.png) will take you to the project in Toloka.
+   - ![Open Project in CVAT](/images/toloka05.png) will take you back to the project in CVAT.
+
+In CVAT, all projects related to Toloka will be labeled as **Toloka**.
+
+![Toloka Label on Project](/images/toloka06.png)
+
+## Adding tasks and jobs to the Toloka project
+
+To add tasks to the Toloka project,
+see [**Create annotation task**](/docs/manual/basics/create_an_annotation_task/#create-a-task).
+
+On step 2 of the **Create annotation task** procedure,
+from the drop-down list, select your Toloka project.
+
+![Toloka Add Task](/images/toloka07.jpg)
+
+## Adding instructions for annotators to the Toloka project
+
+To add instructions for annotators to the Toloka project,
+see [**Adding specification to Project**](/docs/manual/advanced/specification/#adding-specification-to-project)
+documentation or [**Adding instructions**](https://www.youtube.com/embed/hAN9UGRvwOk)
+video tutorial.
+
+## Toloka pool setup
+
+After creating a task and its associated jobs in
+CVAT, you'll need to configure a Toloka pool,
+specifying all task requirements and setting the price for task completion.
+
+To set up the Toloka pool, do the following:
+
+1. Open Toloka task, go to **Actions** > **Setup Toloka pool**.
+
+   ![Set up Toloka pool](/images/toloka08.jpg)
+
+2. In the **Create crowdsourcing task** form, fill in the following fields:
+
+   ![Set up Toloka pool](/images/toloka09.jpg)
+
+   - **Price per job**: Specify the payment amount for completing
+     one ощи within the project.
+   - **Use project description**: Switch this toggle if you want
+     to use the overarching project description for individual tasks.
+   - **Description**: Provide details about the project.
+     This field is visible only when the **Use project description** toggle is off.
+   - **Sensible content**: Switch this toggle if your dataset
+     contains images intended for an adult audience.
+   - **Accept solutions automatically**: Enable this if you
+     wish for completed jobs to be automatically accepted.
+   - **Close pool after completion, sec**: Enter the time limit, in seconds,
+     within which each job should be completed. You can also click on the
+     field to select from one of the predefined periods.
```

Comment:
Wrong description.

The interval during which the pool will remain open from the moment all tasks are completed.
Minimum — 0, maximum — 259200 seconds (three days).